test/commands/lwc/test/run.test.ts

```diff
@@ -11,6 +11,7 @@ import { expect, test } from '@salesforce/command/lib/test';
 import { testSetup } from '@salesforce/core/lib/testSetup';
 import { stubMethod } from '@salesforce/ts-sinon';
 
+// eslint-disable-next-line
```

Comment:
why `lib` and not `src` ?

Reply:
That is what they were, when I change them to `src` test start failing. I only noticed it because of the new(ish) linter rule that came in when i bumped `eslint-config-salesforce`. 

Honestly didn't want to waste time trying to figure out why there were failing. I can poke at it some more now that we are delaying node 20 a week